packs/nitropack.js

```diff
@@ -0,0 +1,41 @@
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Minify resources` is the name of a feature and becomes link text to additional documentation.*/
+  'unminified-css': 'Enable [`Minify resources`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources) in your Caching settings to reduce the size of your CSS, HTML, and JavaScript files for faster load times. This will remove any spaces or comments from your files without changing it\'s functionality.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Minify resources` is the name of a feature and becomes link text to additional documentation.*/
+  'unminified-javascript': 'Enable [`Minify resources`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources) in your Caching settings to reduce the size of your JS, HTML, and CSS files for faster load times. This will remove any spaces or comments from your files without changing it\'s functionality.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `<FILL>`. This is displayed after a user expands the section to see more. No character length limits. `Delayed Scripts` and `Automatic Image Lazy Loading` are names of features and become links to additional documentation.*/
+  'unused-javascript': 'Not all JavaScript code is needed at all times. Configure [`Delayed Scripts`](https://support.nitropack.io/hc/en-us/articles/1500002600942-Delayed-Scripts) in NitroPack to delay loading of such scripts until they are needed and fix this warning. Furthermore, activate [`Automatic Image Lazy Loading`](https://support.nitropack.io/hc/en-us/articles/12457493524369-NitroPack-Lazy-Loading-Feature-for-Images) which will automatically detect iframes and video embeds, and load their JavaScript only when needed.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Improve Server Response Time` is the name of a feature and becomes link text to additional documentation.*/
+  'uses-long-cache-ttl': 'Go to the [`Improve Server Response Time`](https://support.nitropack.io/hc/en-us/articles/1500002321821-Improve-Server-Response-Time) feature in the Caching menu and adjust your page cache expiration time to improve loading times and user experience.',
```

Comment:
Is "Caching menu" the name of a feature in nitropack? If so can you surround it with backticks so that translators know not to translate it?